### PR TITLE
The Incredible Fusion of Medical HUD and Prescription Glasses

### PR DESCRIPTION
**Pull request by Vexylius**
## About The Pull Request

Medical Doctors with poor eyesight - rejoice! Today our R&D team came up with an idea to integrate a medical HUD in a pair of  prescription glasses... and they failed! So the next thing they tried was putting prescription lenses in an abomination made out of two medical HUDs, and that attempt was more successful.
The result looks more-or-less like this:
image
(Credits to @Naevii for the sprite (that I've slightly changed))

Available after researching HUDs in a medical lathe and as a loadout item for  a price of 3500 beecoins.

## Why It's Good For The Game

If you can put medhud on sunglasses, why not on prescription glasses?

## Changelog
:cl:
add: Prescription Medical HUD
/:cl:

### CODE REVIEW

File: code/modules/client/loadout/loadout_accessories.dm [hunk 1]
```diff
@@ -147,6 +147,12 @@
 	display_name = "red glasses"
 	path = /obj/item/clothing/glasses/red
 
+/datum/gear/accessory/glasses/hud/health/prescription
+	display_name = "prescription medical HUD"
+	path = /obj/item/clothing/glasses/hud/health/prescription
+	allowed_roles = list("Paramedic", "Brig Physician", "Medical Doctor", "Chief Medical Officer")
+	cost = 3500
```
Comment (Archanial) [inline code/modules/client/loadout/loadout_accessories.dm]:
```suggestion
```
no free huds from beeshop

**Review by Archanial — CHANGES_REQUESTED** [review]

**Comment by RDS88-beeman** [comment]
hey fun idea what if instead of giving nearsighted brig physes a medHUD implant give them this

File: code/modules/client/loadout/loadout_accessories.dm (hunk 1 above)
Comment (Vexylius) [reply code/modules/client/loadout/loadout_accessories.dm]: aight

**Comment by BrumalLake** [comment]
> 
> 
> hey fun idea what if instead of giving nearsighted brig physes a medHUD implant give them this

i don't remember brig physes getting that

**Comment by super12pl** [comment]
> hey fun idea what if instead of giving nearsighted brig physes a medHUD implant give them this

CMO was the one with medHUD implant dummy
What maining xenobio does to a man

**Comment by imsynthex** [comment]
> CMO was the one with medHUD implant dummy

nearsighted brig phys gets a bugged medhud implant that can't be removed and doesn't show up on scanners :)

**Comment by super12pl** [comment]
> > CMO was the one with medHUD implant dummy
> 
> nearsighted brig phys gets a bugged medhud implant that can't be removed and doesn't show up on scanners :)

what the fuck

**Comment by BrumalLake** [comment]
probably not an implant then, just puts the hudglasses on then deletes them without removing their hud element

**Review by Archanial — APPROVED** [review]

**Pull request merged**